docs/features/software-catalog/descriptor-format.md

```diff
@@ -353,6 +355,7 @@ Fields of a link are:
 | `url`   | String | [Required] A `url` in a standard `uri` format (e.g. `https://example.com/some/page`) |
 | `title` | String | [Optional] A user friendly display name for the link.                                |
 | `icon`  | String | [Optional] A key representing a visual icon to be displayed in the UI.               |
+| `type`  | String | [Optional] An optional value to categorize links into specific groups.               |
```

Comment:
see the "note" block below - it might be good to add a new paragraph below that one, briefly saying what the semantics of the `type` field is. Something that mentions "The semantics of the `type` field are undefined, and the adopter is free to choose what set of values to allow here and how to make use of them in the consuming end." ... not a great wording, just meant as inspiration